tools/InstallToVSCode/InstallToVSCode.sh

```diff
@@ -199,8 +198,31 @@ hash dotnet 2>/dev/null
 SetupSymLink "$CSharpExtensionRoot/coreclr-debug/debugAdapters" "$DESTDIR"
 [ $? -ne 0 ] && echo "ERROR: Unable to link $CSharpExtensionRoot/coreclr-debug/debugAdapters to $DESTDIR" && exit 1
 
-pushd $script_dir/CLRDependencies 1>/dev/null 2>/dev/null
-[ $? -ne 0 ] && echo "ERROR: Unable to find CLRDependencies directory???" && exit 1
+mkdir -p "$DESTDIR/CLRDependencies"
+[ $? -ne 0 ] && echo "ERROR: unable to create destination directory '$DESTDIR/CLRDependencies'." && exit 1
+
+cp -r $script_dir/CLRDependencies/* $DESTDIR/CLRDependencies
+[ $? -ne 0 ] && echo "ERROR: unable to create destination copy CLRDependencies directory." && exit 1
+
+pushd $DESTDIR/CLRDependencies 1>/dev/null 2>/dev/null
+[ $? -ne 0 ] && echo "ERROR: Unable to change to CLRDependencies directory???" && exit 1
+
+OSName=$(uname -s)
+if [ "$OSName" == "Darwin" ]; then
+    # On OSX, hard code the runtime id so we don't need to worry about version.
+    runtime_id="osx.10.11-x64"
```

Comment:
Do we know if the CLI team plans on removing the --infer-runtimes switch? That brings back the old behavior.